src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -831,6 +837,74 @@ class Browser extends DashboardView {
     });
   }
 
+  showExportSelectedRowsDialog(rows) {
+    this.setState({
+      rowsToExport: rows
+    });
+  }
+
+  cancelExportSelectedRows() {
+    this.setState({
+      rowsToExport: null
+    });
+  }
+
+  async confirmExportSelectedRows(rows) {
+    this.setState({ rowsToExport: null });
+    const className = this.props.params.className;
+    const query = new Parse.Query(className);
+
+    if (rows['*']) {
+      // Export all
+      query.limit(10000);
+    } else {
+      // Export selected
+      const objectIds = [];
+      for (const objectId in this.state.rowsToExport) {
+        objectIds.push(objectId);
+      }
+      query.containedIn('objectId', objectIds);
+    }
+
+    const classColumns = this.getClassColumns(className, false);
+    // create object with classColumns as property keys needed for ColumnPreferences.getOrder function
+    const columnsObject = {};
+    classColumns.forEach((column) => {
+      columnsObject[column.name] = column;
+    });
+    // get ordered list of class columns
+    const columns = ColumnPreferences.getOrder(
+      columnsObject,
+      this.context.currentApp.applicationId,
+      className
+    ).filter(column => column.visible);
+
+    const objects = await query.find({ useMasterKey: true });
+    let csvString = columns.map(column => column.name).join(',') + '\n';
+    for (const object of objects) {
+      const row = columns.map(column => {
+        const type = columnsObject[column.name].type;
+        if (column.name === 'objectId') {
+          return object.id;
+        } else if (type === 'Relation' || type === 'Pointer') {
+          return object.get(column.name).id;
+        } else {
```

Comment:
```suggestion
        } else if (type === 'Relation' || type === 'Pointer') {
          if (object.get(column.name)) {
          return  object.get(column.name).id
          } else {
          return 'undefined'
          }; 
        } else {
```
This should fix the issue with the empty pointers.

Reply:
Hey @Vortec4800, is this suggestion okay?

Reply:
Sorry I haven't had a lot of time to dedicate to this lately, but I do at the moment so let me run through these suggestions and changes and try to get them all merged together.